templates/cloud/index.html

```diff
@@ -1,250 +1,364 @@
+            <ul class="p-list--divided">
+              <li class="p-list__item is-crossed">
+                Purchasing and provisioning new hardware carries relatively high fixed costs, which are wasted if the cloud has low utilisation.
+              </li>
+              <li class="p-list__item is-crossed">
+                The deployment and on-going management of a private cloud requires dedicated team resources and technical skill.
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row--50-50">
+      <hr class="is-muted" />
+      <div class="col"></div>
+      <div class="col">
+        <p>
+          <a href="/engage/openstack-introduction">Read an introduction to OpenStack&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+  </section>
 
-<section class="p-strip">
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <h2>What is a public cloud?</h2>
-      <p>In a <a href="/cloud/public-cloud">public cloud environment</a>, compute, storage and other infrastructure resources are provided as a service by an external provider.</p>
-    </div>
-    <div class="col-4 u-hide--small u-vertically-center u-align--center">
-      {{
-        image(
-         url="https://assets.ubuntu.com/v1/a3694a61-Public+Cloud.svg",
-         alt="Public cloud",
-         width="190",
-         height="130",
-         hi_def=True,
-         loading="auto",
-        ) | safe
-      }}  
-    </div>
-  </div>
-  <div class="row u-equal-height">
-    <div class="col-5">
-      <p><b>Advantages:</b></p>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">Immediate access to infrastructure as a service (IaaS) can significantly accelerate time to market for organisations.</li>
-        <li class="p-list__item is-ticked">Public clouds provide the ability to scale resource consumption as needed and virtually without limits.</li>
-      </ul>
-    </div>
-    <div class="col-5">
-      <p><b>Disadvantages:</b></p>
-      <ul class="p-list">
-        <li class="p-list__item is-crossed"><a href="https://go.451research.com/Emerging-pricing-model-for-private-cloud">Can be more expensive</a> than an efficiently-managed and well-utilised private cloud.</li>
-        <li class="p-list__item is-crossed">In addition to the compute costs, there can be unexpected ancillary costs like egress (taking your data out of the cloud)</li>
-      </ul>
-    </div>
-  </div>
-  <div class="u-fixed-width">
-    <p>Major public cloud providers include <a href="/aws">Amazon Web Services (AWS)</a>, <a href="/azure">Microsoft Azure</a>, <a href="/gcp">Google Cloud Platform</a>, Oracle and IBM Cloud.</p>
-    <p>
-      <a href="/download/cloud">Learn more about Ubuntu on Public Clouds&nbsp;&rsaquo;</a>
-    </p>
-  </div>
-</section>
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <hr class="p-rule" />
```

Comment:
```suggestion
        <hr />
```
We are going to deprecate the use of `p-rule`, so you can omit it